tests/lib/snaps/test-snapd-network-status-consumer/consumer.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import dbus
+import os
+import sys
+
+def _get_obj():
+    return dbus.SystemBus().get_object("com.ubuntu.connectivity1.NetworkingStatus", "/com/ubuntu/connectivity1/NetworkingStatus")
+
+def get_version():
+    obj = _get_obj()
+    print(obj.GetVersion(dbus_interface="com.ubuntu.connectivity1.NetworkingStatus"))
```

Comment:
Why is this printing the version instead of returning it?

Reply:
Thanks for reviewing this. 
It is printing because then we match that text in the test. Otherwise I need to print it in the main function.